pkg/util/validation/notifications_limit_flag.go

```diff
@@ -0,0 +1,57 @@
+package validation
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/cortexproject/cortex/pkg/util"
+)
+
+// When changing this, please update documentation for Limits.NotificationIntegrationLimits field.
```

Comment:
Why we do ned to update the doc for  Limits.NotificationIntegrationLimits field? The list of integration names is auto-generated there.